catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/ui/regression/activitydestroy/LegoConfigDialogActivityRecreationRegressionTest.java

```diff
@@ -1,36 +1,36 @@
 /*
-* Catroid: An on-device visual programming system for Android devices
-* Copyright (C) 2010-2018 The Catrobat Team
-* (<http://developer.catrobat.org/credits>)
-*
-* This program is free software: you can redistribute it and/or modify
-* it under the terms of the GNU Affero General Public License as
-* published by the Free Software Foundation, either version 3 of the
-* License, or (at your option) any later version.
-*
-* An additional term exception under section 7 of the GNU Affero
-* General Public License, version 3, is available at
-* http://developer.catrobat.org/license_additional_term
-*
-* This program is distributed in the hope that it will be useful,
-* but WITHOUT ANY WARRANTY; without even the implied warranty of
-* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
-* GNU Affero General Public License for more details.
-*
-* You should have received a copy of the GNU Affero General Public License
-* along with this program.  If not, see <http://www.gnu.org/licenses/>.
-*/
+ * Catroid: An on-device visual programming system for Android devices
```

Comment:
please remove spaces

Reply:
why didn't any of the static checks catch this?

Reply:
checked it again, seems to be a GitHub issue. there is no change.

Reply:
Checkstyle checks against a [license template (agpl_license_text.txt)](https://github.com/Catrobat/Catroid/blob/5a1cf1e299953a9f1918ec96cc3034260252419e/catroid/src/test/res/agpl_license_text.txt) which is written generally to work with multiple file types (java, xml, gradle). To catch this, we could be more strict by either

- creating templates for each file type.
- adding a space to the beginning of each line. java and gradle files are prefixed with a single space, xml with two spaces.
```diff
- ^\W*Catroid
+ ^ \W*Catroid
```
- or be as strict as possible for java, gradle and xml files
```diff
- ^\W*Catroid
+ ^((  ~)|( \*)) Catroid
```

Reply:
…but since this hardly ever happens, we don't need to adapt it for now imo. When creating new java/xml files with Android Studio, the header is automatically added and formatted correctly. If this happens more often, let me know and I'll adapt Checkstyle accordingly.

Mark conversation as resolved because

> seems to be the other way around: there were no spaces where there should have been spaces. so the changes do the correct thing.